### ISSUE TITLE
Please support concertgebouw.videodock.com

### ISSUE THREAD

**Issue by jdietterich** [ytdl-org/youtube-dl#26673]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2020.09.20**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://concertgebouw.videodock.com/view/94
- Single video: http://concertgebouw.videodock.com/view/61

## Description

Site posts free concerts.  No credentials required.

**Comment by chettriyuvraj** [comment]
Hey! I'm new here and would like to try solving this request.

**Comment by jdietterich** [comment]
Wonderful!  Would be most grateful.

**Comment by NateChoe1** [comment]
Does videodock have something other than concertgebouw.videodock.com? Or are all videos on that domain?

**Comment by hetship** [comment]
Yes.  It is my understanding that videodock.com hosts other videos as well.  Thank you.

**Comment by hetship** [comment]
You can find links to concert videos hosted by videodock.com on this web page:  https://www.concertgebouw.nl/en/watch-free-live-streams.  Just click on any of the yellow boxes that say "LIVE HD".

**Comment by grexe** [comment]
is there any progress on this? I've just discovered this wonderful site and would like to watch the videos on my TV, fed from my media server...
e.g. https://concertgebouw.videodock.com/view/96
